_posts/2020-06-29-adoptopenjdk-eclipse.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  'AdoptOpenJDK becomes Eclipse Adoptium'
+author: hendrik
+featuredImage: comment
+excerpt: 'A few days ago, AdoptOpenJDK officially announced its accession to the Eclipse Foundation and the future name "Eclipse Adoptium". A necessary and sensible move? A comment by Hendrik Ebbers, member of the Technical Steering Committee of AdoptOpenJDK.' 
+categories: [Java, OpenJDK, Eclipse, Adoptium]
+header:
+  text: AdoptOpenJDK becomes <span class="my-karakun">Eclipse Adoptium</span>
+  image: post
+---
+
+AdoptOpenJDK has become one of the biggest Java respectively OpenJDK distributions with [more than 165M downloads](https://dash-v2.adoptopenjdk.net/). As a member of the Technical Steering Committee (TSC) of AdoptOpenJDK, I did not only follow the project in the last couple of months but was at least in pieces a vital part of it.
+
+## Why AdoptOpenJDK?
+
+If I was asked 5 years ago about where to download Java I definitely would have referred to the download pages of the Oracle JDK. But in the last couple of years, things have changed. Thanks to OpenJDK, in theory everyone can build its own Java distribution. AND, licensing of Oracle JDK has changed - [it is no longer free to use](/java/2018/06/25/java-releases.html). This is why more and more Java distributions came up and it is sometimes not easy to get a clear overview of this JDK jungle.
+
+![3 Duke Suspects](/assets/images/pages/jdks/3duke_suspects.png)
+
+In contrast to OpenJDK distributions by companies like Oracle, IBM, Azul or Bellsoft, [AdoptOpenJDK](https://adoptopenjdk.net/) is a community-based project providing Java binaries for a large number of platforms. Besides the common binaries for Windows x64, Mac x64 and Linux x64 there are also builds for AIX and ARM.
+
```

Comment:
<img width="1012" alt="platforms" src="https://user-images.githubusercontent.com/9443847/88282414-d323b380-cce9-11ea-8911-a3599eff804c.png">
Image of platforms can be added